ibis/pandas/udf.py

```diff
@@ -604,3 +526,73 @@ def wrapped(*args):
             return wrapped
 
         return wrapper
+
+
+@pre_execute.register(ops.ElementWiseVectorizedUDF)
```

Comment:
this is just a move of the other code (mostly)?

Reply:
Yes mostly move code around.

There is one change - the original code will create a class dynamically here for each UDF. In this PR, this is changed so that UDF share the same Node class (ElementWiseVectorizedUDF) and the node class has a "func" field that contains the actually callable (along with other information about the UDF)